src/tasks/delayedGiveawayCreate.ts

```diff
@@ -0,0 +1,13 @@
+// This task will be created by the gschedule command when a user schedules a giveaway to be created.
```

Comment:
`gschedule` command? We don't have such a command

Reply:
Are you saying there's <!-- a multiverse --> no gschedule command?!?

Out of jokes it's `gstart`, which has an alias `giveawayschedule`, but no `gschedule`:
![image](https://user-images.githubusercontent.com/24852502/87347098-f1075080-c552-11ea-8cba-a1da2d9ec844.png)